extensions/vscode/package.json.in

```diff
@@ -1,128 +1,107 @@
-					"default": "off",
-					"description": "Traces the communication between VS Code and the language server.",
-					"enum": [
-						"off",
-						"messages",
-						"verbose"
-					],
-					"scope": "window",
-					"type": "string"
-				},
-				"wakeTimeline.pathToWakeBinary": {
-					"default": "",
-					"description": "The path to the wake binary that is used to create the timeline. If invalid, the timeline functionality is not provided.",
-					"scope": "window",
-					"type": "string"
-				}
-			}
-		},
-		"grammars": [
-			{
-				"language": "wake",
-				"path": "./syntaxes/wake.tmLanguage.json",
-				"scopeName": "source.wake"
-			}
-		],
-		"languages": [
-			{
-				"aliases": [
-					"wake",
-					"wake"
-				],
-				"configuration": "./language-configuration.json",
-				"extensions": [
-					".wake"
-				],
-				"id": "wake"
-			}
-		],
-		"commands": [
-			{
-				"command": "timeline.create",
-				"title": "Create",
-				"category": "Timeline"
-			},
-			{
-				"command": "timeline.refresh",
-				"title": "Refresh",
-				"category": "Timeline"
-			}
-		]
-	},
-	"scripts": {
-		"vscode:prepublish": "npm run package",
-		"package": "webpack --mode production --devtool hidden-source-map",
-		"compile": "webpack",
-		"watch": "webpack --watch",
-		"open-in-browser": "vscode-test-web --browserType=chromium --extensionDevelopmentPath=."
-	},
-	"dependencies": {
-		"vscode-languageclient": "^7.0.0",
-		"vscode-languageserver": "^8.0.1",
-		"ts-mutex": "^1.0.0"
-	},
-	"devDependencies": {
-		"@types/node": "^15.14.9",
-		"@types/vscode": "^1.70.0",
-		"@types/webpack-env": "^1.17.0",
-		"@vscode/test-web": "^0.0.24",
-		"eslint": "^7.32.0",
-		"path-browserify": "^1.0.1",
-		"process": "^0.11.10",
-		"util": "^0.12.4",
-		"crypto-browserify": "^3.12.0",
-		"ts-loader": "^9.3.0",
-		"typescript": "^4.4.4",
-		"webpack": "^5.72.1",
-		"webpack-cli": "^4.9.2",
-		"@types/emscripten": "^1.39.5"
-	}
+  "name": "wake",
+  "displayName": "Wake",
+  "description": "A language server for the SiFive wake build tool",
+  "icon": "images/sifive_icon.png",
+  "author": "SiFive",
+  "license": "Apache-2.0",
+  "version": "@VERSION@",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/sifive/wake.git"
+  },
+  "bugs": {
+    "url": "https://github.com/sifive/wake/issues"
+  },
+  "publisher": "SiFive",
+  "engines": {
+    "vscode": "^1.92.0"
+  },
+  "categories": [
+    "Programming Languages"
+  ],
+  "keywords": [
+    "wake",
+    "build tool",
+    "sifive"
+  ],
+  "activationEvents": [
+    "onLanguage:wake"
+  ],
+  "main": "./dist/extension.js",
+  "contributes": {
+    "configuration": {
+      "id": "wakeLanguageServer",
+      "title": "Wake",
+      "type": "object",
+      "properties": {
+        "wakeLanguageServer.standardLibrary": {
+          "default": null,
+          "description": "The path to the wake standard library. By default the version bundled with wake is used.",
```

Comment:
Can we mention that the "the version bundled with wake" is coming from the wake specified by the `path` property?